src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceWorkExperienceReferences.vue

```diff
@@ -216,7 +216,7 @@ export default defineComponent({
           lastName: this.lastName,
           emailAddress: this.email,
           phoneNumber: this.phoneNumber,
-          hours: parseInt(this.hours),
+          hours: parseInt(this.hours!.toString()),
```

Comment:
Do we need to do this conversion? 

I thought v-model.isNumber on line 43

<v-text-field
          v-model.number="hours"
          
          would parse it to a number already. 
          
          https://vuejs.org/guide/essentials/forms#number